### PR TITLE
Create Sponsorship Manager Role with Resume Permissions

### LINKED ISSUE #370: Sponsorship Access Type
Sponsorship board members are typically in charge of the ACM member resume bank, and as such, should have a specialized role on the Portal to access/deal with all visible resumes. At current only Admins are able to view all visible resumes.

### PR DESCRIPTION
**Pull request by yimmyj**
Closes #370 .

### CODE REVIEW

File: tests/resume.test.ts [hunk 1]
```diff
@@ -55,6 +55,37 @@ describe('resume fetching', () => {
     });
   });
 
+  test('sponsorship members can get all visible resumes', async () => {
```
Comment (nik-dange) [inline tests/resume.test.ts:58]: nit: change to sponsorship managers

File: tests/resume.test.ts [hunk 2]
```diff
@@ -55,6 +55,37 @@ describe('resume fetching', () => {
...
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.SPONSORSHIP_MANAGER });
```
Comment (nik-dange) [inline tests/resume.test.ts:60]: nit: rename admin to something like `sponsorshipManager` for readability

**Review by nik-dange — CHANGES_REQUESTED** [review]
Just two tiny nits, otherwise LGTM 🚀

**Comment by nik-dange** [comment]
Also, make sure to add a seeding user with this role for the testing db

**Comment by dowhep** [comment]
Remind to rename this PR as well

**Comment by yimmyj** [comment]
Logged in with newly created sponsorship manager role
Screenshot 2023-12-25 at 3 12 44 PM

Retrieved resumes without error
Screenshot 2023-12-25 at 3 13 04 PM

**Review by nik-dange — APPROVED** [review]
LGTM 🏋️

**Pull request merged**